File: online-judge-solutions/Codeforces/1700A.py
#https://codeforces.com/problemset/problem/1700/A
#Lang		:	Python 3.8
#Time		:	62 ms
#Memory         :	0 KB
for _ in range(int(input())):
    n,m=map(int,input().split())
    print(int((m*(m+1))/2+m*((n*(n+1))/2)-m))

File: online-judge-solutions/Codeforces/1700A.cpp
//https://codeforces.com/problemset/problem/1700/A
//Lang		:	GNU C++14
//Time		:	30 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int t;
	long long n,m;
	cin>>t;
	while(t--){
		cin>>n>>m;
		cout<<(m*(m+1))/2+m*((n*(n+1))/2)-m<<endl;
	}
	return 0;
}
